tests/phpunit/CRM/Export/BAO/ExportTest.php

```diff
@@ -129,6 +130,20 @@ public function testExportComponentsContribution() {
       'exportMode' => CRM_Export_Form_Select::CONTRIBUTE_EXPORT,
       'componentClause' => 'civicrm_contribution.id IN ( ' . implode(',', $this->contributionIDs) . ')',
     ]);
+    $this->assertContains('display', array_values($this->csv->getHeader()));
+    $row = $this->csv->fetchOne(0);
+    $this->assertEquals('This is a test', $row['display']);
+  }
+
+  /**
+   * Implements hook_civicrm_export().
+   *
+   */
+  public function confirmHookWasCalled(&$exportTempTable, &$headerRows, &$sqlColumns, $exportMode, $componentTable, $ids) {
+    $sqlColumns['display'] = 'dipslay varchar(255)';
```

Comment:
@seamuslee001 Is the `dipslay` typo on purpose - shouldn't that make something fail? Ignore me if I'm just misunderstanding this test.

Reply:
hmm i don't think it was meant to fail as i think the header rows does the trick and i have essentially compensated in L145 but i will fix this up anyway